src/adapter/http/src/data/account_handler.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use axum::extract::Extension;
+use axum::response::Json;
+use chrono::{DateTime, Utc};
+use database_common_macros::transactional_handler;
+use dill::Catalog;
+use http_common::*;
+use kamu_accounts::{
+    Account,
+    AccountDisplayName,
+    AccountType,
+    AuthenticationService,
+    CurrentAccountSubject,
+};
+use opendatafabric::{AccountID, AccountName};
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#[derive(Debug, serde::Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AccountResponse {
+    pub id: AccountID,
+    pub account_name: AccountName,
+    pub email: Option<String>,
+    pub display_name: AccountDisplayName,
+    pub account_type: AccountType,
+    pub avatar_url: Option<String>,
+    pub registered_at: DateTime<Utc>,
+    pub is_admin: bool,
+    pub provider: String,
+    pub provider_identity_key: String,
+}
+
```

Comment:
Do we really need to explose this entire struct? For your task you would probably be fine with `id` and `account_name`, right?